dashboard/settings.py

```diff
@@ -115,19 +112,38 @@
 # Static files (CSS, JavaScript, Images)
 # https://docs.djangoproject.com/en/1.9/howto/static-files/
 
-BASE_DIR = os.path.dirname(os.path.abspath(__file__))
+PROJECT_DIR = os.path.dirname(os.path.abspath(__file__))
```

Comment:
@openjck note: I'm mostly :neutral_face: on it, but I changed the name of this to `PROJECT_DIR` since it's technically in the `dashboard` project directory - not the base of the repository.

Unless there's a `django-pipeline` convention, if we're going to use these static libs in other templates (e.g., domains app templates), maybe we'd rather put them in the repository `BASE_DIR`? So `static/lib` instead of `dashboard/static/lib`?